lumen/ai/models/details.js

```diff
@@ -0,0 +1,34 @@
+export function render({ model, view }) {
```

Comment:
Since TableLookup output both text and schemas, the schema took up lots of space in the ChatStep, and it wasn't formatted well, so I created this Details component to hide it.


<img width="696" alt="image" src="https://github.com/user-attachments/assets/66828551-92d2-4c8d-a539-f004daf0615d" />

<img width="650" alt="image" src="https://github.com/user-attachments/assets/1895216c-8351-4d96-94ef-16bbbb05fdb8" />